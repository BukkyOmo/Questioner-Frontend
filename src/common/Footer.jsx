import React, { Component } from 'react';

const Footer = () => (
  <footer>
    <p>&copy;2018 All Rights Reserved</p>
    <p>Website Designed by Bukola Omosefunmi</p>
  </footer>
);

export default Footer;
