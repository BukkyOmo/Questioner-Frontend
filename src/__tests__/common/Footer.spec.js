import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../common/Footer';

describe('<Footer />', () => {
  it('renders h1 element', () => {
    const wrapper = shallow(<Footer />)
    expect(wrapper.find('p').exists()).toBe(true);
  });
   it('renders correctly using snapshot', () => {
    expect(Footer).toMatchSnapshot();
   });
});
