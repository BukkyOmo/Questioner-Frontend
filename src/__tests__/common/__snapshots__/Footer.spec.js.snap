// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders correctly using snapshot 1`] = `[Function]`;
